lib/calculator.rb

```diff
@@ -0,0 +1,29 @@
+class Calculator
+
+    def self.add_values(current_frame, next_frame, third_frame, turn)
+        total = current_frame[0]
```

Comment:
lets also be more specific with what total this is. Is this a total for the frame? For the game?